exporters/elasticsearch-exporter/README.md

```diff
@@ -1,172 +1,3 @@
-
-More specifically, each option configures the following:
-
-* `prefix` (`string`): this prefix will be appended to every index created by the exporter; must not
-  contain `_` (underscore).
-* `createTemplate` (`boolean`): if true, missing indexes will be created as needed.
-* `command` (`boolean`): if true, command records will be exported; if false, ignored.
-* `event` (`boolean`): if true, event records will be exported; if false, ignored.
-* `rejection` (`boolean`): if true, rejection records will be exported; if false, ignored.
-* `deployment` (`boolean`): if true, records related to deployments will be exported; if false,
-  ignored.
-* `incident` (`boolean`): if true, records related to incidents will be exported; if false, ignored.
-* `job` (`boolean`): if true, records related to jobs will be exported; if false, ignored.
-* `jobBatch` (`boolean`): if true, records related to job batches will be exported; if false,
-  ignored.
-* `message` (`boolean`): if true, records related to messages will be exported; if false, ignored.
-* `messageSubscription` (`boolean`): if true, records related to message subscriptions will be
-  exported; if false, ignored.
-* `processInstance` (`boolean`): if true, records related to process instances will be exported; if
-  false, ignored.
-* `processMessageSubscription` (`boolean`): if true, records related to process message
-  subscriptions will be exported; if false, ignored.
-
-Here is a complete, default configuration example:
-
-```yaml
-...
-exporters:
-  elasticsearch:
-    # Elasticsearch Exporter ----------
-    # An example configuration for the elasticsearch exporter:
-    #
-    # These setting can also be overridden using the environment variables "ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_..."
-    #
-
-    className: io.camunda.zeebe.exporter.ElasticsearchExporter
-
-    args:
-      # A comma separated list of URLs pointing to the Elasticsearch instances you wish to export to.
-      # For example, if you want to connect to multiple nodes for redundancy:
-      # url: http://localhost:9200,http://localhost:9201
-      url: http://localhost:9200
-
-      bulk:
-        delay: 5
-        size: 1000
-        memoryLimit: 10485760
-
-      authentication:
-        username: elastic
-        password: changeme
-
-      index:
-        prefix: zeebe-record
-        createTemplate: true
-
-        command: false
-        event: true
-        rejection: false
-
-        deployment: false
-        process: true
-        error: true
-        incident: true
-        job: true
-        jobBatch: false
-        message: false
-        messageSubscription: false
-        variable: true
-        variableDocument: true
-        processInstance: true
-        processInstanceCreation: false
-        processMessageSubscription: false
-```
-
+Please refer to the [official documentation](https://docs.camunda.io/docs/next/self-managed/zeebe-deployment/exporters/elasticsearch-exporter/)
```

Comment:
:question: Is it okay that we're linking to the `next` version of the docs on the `stable/8.1` branch? 

It's possible that we may misinform users that follow the link. And we don't have the "Exporters" doc section in the 8.1 docs.

Reply:
Same as the other PR, since we don't have the exporters in the docs for this version we shouldn't backport this.